coreSDK/amd/package.json

```diff
@@ -1,14 +1,10 @@
 {
     "name": "applicationinsights-core-js",
-    "version": "0.0.15",
+    "version": "0.0.37",
     "description": "Microsoft Application Insights Core Javascript SDK",
     "main": "bundle/applicationinsights-core-js.js",
     "types": "bundle/applicationinsights-core-js.d.ts",
-    "devDependencies": {
-        "grunt": "1.0.1",
```

Comment:
>grunt [](start = 9, length = 5)

are you removing the grunt dependency because you're assuming the grunt binary was installed from the root directory?

Reply:
We chatted about this, grunt is a dependency in the build system that we have and not in the package that is shipped.